dipy/reconst/shm.py

```diff
@@ -904,7 +1069,7 @@ def sf_to_sh(sf, sphere, sh_order=4, basis_type=None, smooth=0.0):
     return sh
 
 
-def sh_to_sf(sh, sphere, sh_order, basis_type=None):
+def sh_to_sf(sh, sphere, sh_order=4, basis_type=None):
```

Comment:
Not sure why you set default to 4 here.  Can you explain?

Reply:
Hi. The function header said the `sh_order`'s default value was 4, but the function's signature didn't fit the description. I could have changed the comment instead of the function signature.